### PR TITLE
[feat] single line string validation

### PR DESCRIPTION
**Pull request by erfanium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -916,7 +916,8 @@ Property | Default  | Description
 `numeric`   | `null`   | The value must be a numeric string.
 `alphanum`   | `null`   | The value must be an alphanumeric string.
 `alphadash`   | `null`   | The value must be an alphabetic string that contains dashes.
-`hex`   | `null`   | The value must be a hex string.
+`hex`   | `null`   | The value must be a single line string.
+`singleLine`   | `null`   | The value must be a hex string.
```
Comment (icebob) [inline README.md:920]: The descriptions are swapped.

Comment (erfanium) [reply README.md]: Fixed✅

**Comment by erfanium** [comment]
@icebob Thanks, Many new features have been added since the previous release, When's the next release time?

**Comment by icebob** [comment]
I will try in the next days. _I've just arrived from a holiday and I'm a little bit busy._

**Comment by icebob** [comment]
Released: https://github.com/icebob/fastest-validator/releases/tag/v1.6.0

**Pull request merged**